splinepy/multipatch.py

```diff
@@ -0,0 +1,358 @@
+"""
+Multipatch Spline Configuration
+"""
+
+import numpy as np
+
+from splinepy.utils import log
+
+
+class Multipatch():
```

Comment:
```suggestion
class Multipatch(SplinepyBase):
```
you could start using `self._log*`